src/ShopBundle/Resources/config/services.xml

```diff
@@ -276,6 +276,17 @@
         <service id="chameleon_system_shop.product_variant_service" class="ChameleonSystem\ShopBundle\Service\ProductVariantService">
         </service>
 
+        <service
+                id="chameleon_system_shop.basket.basket_variable_replacer"
+                class="ChameleonSystem\ShopBundle\Basket\BasketVariableReplacer"
+        >
+            <argument type="service" id="request_stack"/>
+            <argument type="service" id="twig"/>
+            <argument type="service" id="logger" on-invalid="ignore"/>
```

Comment:
When could it be invalid?

Reply:
I shouldn't. But as a project is free to configure the logger however it pleases we may as well make sure.